packages/web-components/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@department-of-veterans-affairs/web-components",
-  "version": "0.22.0",
+  "version": "1.0.0",
```

Comment:
Major version 0 is a bit special since it indicates that nothing should be considered fully stable. Maybe it's finally time for us to make that bump even without this `close` thing.

Reply:
I didn't know that but thanks! I can revert this if you'd like.

Reply:
I just made a note for us to discuss what kind of guidelines we want to set for versioning this package. Since this would be a breaking change, I am comfortable with bumping the package to 1.0.0 as long as you are OK with it @bkjohnson.

Reply:
I'm fine with it! [Semver guidelines](https://semver.org/#how-do-i-know-when-to-release-100) suggest that we should already be in 1.0.0:

> If your software is being used in production, it should probably already be 1.0.0. If you have a stable API on which users have come to depend, you should be 1.0.0.

Even though we try to hide the `web-components` package and treat `component-library` like the main package, I think it's good for the version to indicate that we're stable.